bin/templates/template.hbs

```diff
@@ -0,0 +1,21 @@
+{{> header}}
+
+{{#if noteGroups}}
+{{#each noteGroups}}
+#### ⚠ {{title}}
```

Comment:
Is that triangle-and-! character in the space intentionally?

Reply:
(Looks like it might be for breaking changes.)

Reply:
It is for breaking changes, I'm not sure why they do this, but currently the only supported `note` type is a breaking change, so `noteGroups` only has one possible group, which is also breaking change (see https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-commits-parser/test/parser.spec.js for examples of what comes out of the parser)